dotnet/PopcornNetStandard.WebApiCore/ExpandServiceFilter.cs

```diff
@@ -70,6 +70,11 @@ public void OnActionExecuted(ActionExecutedContext context)
             {
                 return;
             }
+            // If there response is not OK(200) it should be an error code.
+            else if (!(context.Result is OkObjectResult))
```

Comment:
I think this is going to match too many things; not every success will be an OkObjectResult.  Perhaps in the case below, can we just copy the HttpStatus code over?  Or if it isn't a success code, return early?

Reply:
Either should work

Reply:
status code will be copied to the new response to preserve it rather than just returning the !OKObjectResult items.